app/src/main/java/ca/llamabagel/transpo/ui/trips/GetNextBusTripsUseCase.kt

```diff
@@ -18,16 +19,16 @@ import kotlinx.coroutines.flow.flowOn
 import kotlinx.coroutines.flow.map
 import javax.inject.Inject
 
-class GetNextBusTripsUseCase @Inject constructor(private val repository: TripsRepository) {
-
+class GetNextBusTripsUseCase @Inject constructor(private val repository: TripsRepository,
```

Comment:
Idk if it's a big deal but TripsRepository should be marked with `@ExperimentalCoroutinesApi`

Reply:
That unlocks a pandora's box of things that need to be marked `@ExperimentalCoroutinesApi`. For correctness I guess, but are we trying to be _that_ correct?